pimd/pim_bsm.c

```diff
@@ -1321,6 +1322,8 @@ int pim_bsm_process(struct interface *ifp, pim_sgaddr *sg, uint8_t *buf,
 	}
 
 	bshdr = (struct bsm_hdr *)(buf + PIM_MSG_HEADER_LEN);
+	pim_inet_dump("<bsr?>", bshdr->bsr_addr.addr, bsr_str, sizeof(bsr_str));
```

Comment:
This doesn't actually fix the problem, it just makes it invisible to clang.

Correct fix is to make a copy of the addr so the compiler makes an aligned copy. Will raise separate fix.